dawn/src/dawn/CodeGen/Cuda-ico/CudaIcoCodeGen.cpp

```diff
@@ -1169,6 +1190,22 @@ static void
 generateF90InterfaceSI(FortranInterfaceModuleGen& fimGen,
                        const std::shared_ptr<iir::StencilInstantiation>& stencilInstantiation) {
   const auto& stencils = stencilInstantiation->getStencils();
+  const auto& globalsMap = stencilInstantiation->getIIR()->getGlobalVariableMap();
+  auto globalTypeToFortType = [](const sir::Global& global) {
+    switch(global.getType()) {
+    case sir::Value::Kind::Boolean:
+      return FortranInterfaceAPI::InterfaceType::CHAR;
```

Comment:
Is it because it's one byte? Does it work?
I think we should add boolean to the interface types.